tests/test_symbol_to_entrez_id.py

```diff
@@ -0,0 +1,48 @@
+import math
+import unittest
```

Comment:
Tests all seem good, but is there a reason you are using `unittest` over `pytest`?

Reply:
No good reason - rewrote tests in `pytest` style and got rid of `unittest` dependency.